apps/storefront/src/pages/QuotesList/index.mobile.test.tsx

```diff
@@ -518,32 +522,10 @@ describe('when the user is a B2C customer', () => {
     });
   });
 
-  describe('when clicking on "view" within the draft quote', () => {
```

Comment:
this was an old version of `'navigates to the quote draft page'` with the incorrect setup (draft coming from the API)

I previously forgot to delete it after I made the new, corrected, version